rwslib/rws_requests/__init__.py

```diff
@@ -525,3 +525,47 @@ def url_path(self):
             args.append(self.formoid)
 
         return make_url(*args, **self._querystring())
+
+
+class ConfigurableDatasetRequest(RWSAuthorizedGetRequest):
+    VALID_DATASET_FORMATS = ()
```

Comment:
Is this good practice?  https://docs.python.org/2/tutorial/classes.html#class-and-instance-variables